src/reader/__tests__/reader.spec.ts

```diff
@@ -270,137 +270,138 @@ var x = true;
       const mdastTree = mdReader.fromLang(input);
 
       expect(prettyStringify(mdastTree)).toMatchInlineSnapshot(`
-        "<>
```

Comment:
Lol, this snapshot decided to change indentation - nothing inside of it actually changed.

Reply:
Isn't it because of the dep bump?

Reply:
Yes that's quite likely